Applications/MmrMobile/Client/MmrMobileClient.cs

```diff
@@ -162,6 +162,17 @@ public async Task<MmrNodeData> GetTimeSeriesFromMachine(string machineNumber, st
         }
         #endregion
 
+        #region machinedata
+
+        /// <inheritdoc />
+        public async Task<IEnumerable<AlertEvent>> GetEventSeriesFromMachine(string machineNumber, DateTime from, DateTime to, int take = 1000, int skip = 0)
```

Comment:
Please do not define a default for take. The client should pass it explicitly so that he is aware of the limit.